pom.xml

```diff
@@ -239,6 +260,14 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<argLine>
+						-javaagent:${settings.localRepository}/org/jmockit/jmockit/${jmockit.version}/jmockit-${jmockit.version}.jar
```

Comment:
Do the newlines around the `<argLine>` tag not create problems here? I guess if it works, then it's fine, but know that whitespace inside XML CDATA is *not* trimmed out intrinsically.